pkg/operation/care/checker.go

```diff
@@ -475,12 +475,13 @@ func (b *HealthChecker) CheckClusterNodes(
 func (b *HealthChecker) CheckMonitoringControlPlane(
 	namespace string,
 	isTestingShoot bool,
+	monitoringEnabled bool,
```

Comment:
Can you combine these two parameters into one `shootMonitoringEnabled` parameter?